src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -50,11 +62,18 @@
     @Mock
     private AppealRepository appealRepository;
 
+    @Mock
+    private ChipsRestClient chipsRestClient;
+
+    @Mock
+    private ChipsConfiguration chipsConfiguration;
+
     @BeforeAll
     public static void beforeTest() throws IOException {
         testAttachments = mapper.readValue(
-            new File("src/test/resources/data/listOfValidAttachments.json"), 
-            new TypeReference<List<Attachment>>() { }
```

Comment:
Was there a reason to remove the reference to the type?

Reply:
Since java 7, the explicit type can be replaced with `<>`

Reply:
Ahh yes type inference